.travis.yml

```diff
@@ -0,0 +1,18 @@
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - vendor
```

Comment:
Is using a local directory here sufficient? In the other .travis.yml files we have made the path explicit: ` $GOPATH/src/github.com/deis/router/vendor`. I've just copied that and assumed it was necessary since I'm unclear what `PWD` is on Travis.